blueprints/app/files/.eslintrc.js

```diff
@@ -43,11 +43,15 @@ module.exports = {
       env: {
         browser: false,
         node: true
-      }<% if (blueprint !== 'app') { %>,
+      },
       plugins: ['node'],
       rules: Object.assign({}, require('eslint-plugin-node').configs.recommended.rules, {
-        // add your custom rules and overrides for node files here
-      })<% } %>
+        // add your custom rules and overrides for node files here<% if (blueprint !== 'app') { %>
```

Comment:
Still has a trailing conditional here, is that intended?

Reply:
Yes. That is the closing `if` to the one above. The fixtures should confirm this is working (once I rebase and get them passing).